authn/api/http/transport.go

```diff
@@ -103,8 +103,8 @@ func encodeError(_ context.Context, err error, w http.ResponseWriter) {
 	switch {
 	case errors.Contains(err, authn.ErrMalformedEntity):
 		w.WriteHeader(http.StatusBadRequest)
-	case errors.Contains(err, authn.ErrUnauthorizedAccess):
-		w.WriteHeader(http.StatusForbidden)
+	case errors.Contains(err, authn.ErrUnauthenticated):
+		w.WriteHeader(http.StatusUnauthorized)
```

Comment:
Should there be 2 cases (in this particular order):
```go
case errors.Contains(err, authn.ErrUnauthenticated):
    w.WriteHeader(http.StatusForbidden)
case errors.Contains(err, authn.ErrUnauthorized):
    w.WriteHeader(http.StatusUnauthorized)
```
The service itself won't return ErrUnauthenticated, but our JWT implementation might.